pipeline_components/aiplatform/aiplatform/update_best_model/component.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 Google LLC
+from typing import NamedTuple
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp.v2.dsl import Input, Model, component
+from pathlib import Path
+
+
+@component(
+    base_image="python:3.7",
+    packages_to_install=["google-cloud-aiplatform>=1.24.1"],
+    output_component_file=str(Path(__file__).with_suffix(".yaml")),
+)
+def update_best_model(
+    challenger: Input[Model],
+    challenger_evaluation: str,
+    parent_model: str,
+    project_id: str,
+    project_location: str,
+    eval_metric: str,
+    eval_lower_is_better: bool,
+) -> NamedTuple("Outputs", [("challenger_wins", bool)]):
+    """
+    Args:
+        challenger (Model): Challenger model.
+        challenger_evaluation (str): Challenger evaluation.
```

Comment:
Can you elaborate on this docstring? i.e. it is the resource name of the Vertex Model Evaluation for the challenger model (and give example of the format)